dask/dataframe/io.py

```diff
@@ -421,6 +422,65 @@ def _pd_to_hdf(pd_to_hdf, lock, args, kwargs=None):
     return None
 
 
+def _pd_to_bcolz(pd_to_bcolz, lock, args, kwargs=None):
+    if lock:
+        lock.acquire()
+    try:    
+        if kwargs['mode']=='w':
+            pd_to_bcolz(args[0], rootdir=args[1], mode='w') # add expectedlen=outfile_Nrows
+        else: 
+            import_ct = bcolz.ctable(rootdir=args[1], mode='a')
+            temp_ct = bcolz.ctable.fromdataframe(args[0])
+            import_ct.append(temp_ct)
```

Comment:
How is this performance-wise? Does it require reading the entire file? Can we optimize this?